test/replay/alloc.scfg

```diff
@@ -0,0 +1,3 @@
+model {
+  symbol symbol_0 i32 9
+}
```

Comment:
It may be better not to store this file and to use something like `owi c alloc.c -o alloc.wasm > alloc.scfg` in the cram test?